.github/workflows/codeql-query.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           CODEQL_ENABLE_EXPERIMENTAL_FEATURES: true
 
       - name: Run query
-        uses: dsp-testing/qc-run2/query@main
+        uses: dsp-testing/qc-run2/query@sarif-snippets
```

Comment:
Remember to flip this back :)

Reply:
Good point! I'll do this at the end, in case we want to test any suggestions/changes before merging. (The PR check will keep me on my toes too 🦶🏽 😅  )

Reply:
I'm inspired! Who even knew that there was a :foot: emoji right there waiting for me to use it?